src/components/Header.jsx

```diff
@@ -1,17 +1,18 @@
 import Link from 'next/link';
 
+import Wrapper from './Positioning/Wrapper';
+
 export default function Header() {
 	return (
 		<header>
 			<nav>
-				<Link href="/">Home</Link>
-				<br />
-				<Link href="/createInvoice">Create Invoice</Link>
+				<Wrapper>
+					<Link href="/">Home</Link>
 
-				<br />
+					<Link href="/createInvoice">Create Invoice</Link>
```

Comment:
Hey Mate, with the Link, you will need an <a> inside to use with next.js
look here https://nextjs.org/docs/api-reference/next/link